s3/src/main/scala/akka/stream/alpakka/s3/scaladsl/S3.scala

```diff
@@ -225,4 +226,72 @@ object S3 {
         chunkSize,
         chunkingParallelism
       )
+
+  /**
+   * Create new bucket with a given name
+   *
+   * @see https://docs.aws.amazon.com/AmazonS3/latest/API/RESTBucketPUT.html
+   *
+   * @param bucketName bucket name
+   * @return [[scala.concurrent.Future Future]] with type [[Done]] as API doesn't return any additional information
+   */
+  def makeBucket(bucketName: String)(implicit mat: Materializer, attr: Attributes = Attributes()): Future[Done] =
```

Comment:
I'm a bit surprised the overload works for picking the correct method based on the return type. And I wonder if it would be better to come up with different names to properly distinguish them.

Reply:
I mean it is kind of good and bad. It is consistent and clear but only works when you specify type.

Do you think that it would be better when `Source` version would be `makeBucket` and future versions: `makeBucketF`?

Reply:
I think I'd prefer to append `Source`. 
```scala
def makeBucketSource(bucketName: String): Source[Done, NotUsed]
``` 
looks OK, doesn't it?

Reply:
Makes sense :+1: 

I will submit a updated version in few hours